scripts/cubic/bootstrap-fmriprep.sh

```diff
@@ -263,24 +306,48 @@ echo "outputsource=${output_store}#$(datalad -f '{infos[dataset][id]}' wtf -S da
 echo "cd ${PROJECTROOT}" >> code/merge_outputs.sh
 wget -qO- ${MERGE_POSTSCRIPT} >> code/merge_outputs.sh
 
+dssource="${input_store}#$(datalad -f '{infos[dataset][id]}' wtf -S dataset)"
+pushgitremote=$(git remote get-url --push output)
```

Comment:
why are these two lines required in the merge script? i also see them at the end of this file.

Reply:
not sure... will try to figure out

Reply:
It's actually not in the merge script. This is only part of the main script and I believe it's not needed here.